code/frameworks/react-webpack5/src/preset.ts

```diff
@@ -9,46 +9,14 @@ export const addons: PresetProperty<'addons'> = [
   getAbsolutePath('@storybook/preset-react-webpack'),
 ];
 
-const defaultFrameworkOptions: FrameworkOptions = {
-  legacyRootApi: true,
-};
-
-export const frameworkOptions = async (
```

Comment:
Did you check whether this code never had any effect, or whether it was necessary at all? Also, this code change makes it so that the PR/release title is something else, and not just "update StorybookConfig import in core-webpack"

Reply:
Well, I rely on the sandboxes.

https://630511d655df72125520f051-pcarsnsigo.chromatic.com/?path=/docs/configure-your-project--docs
https://630511d655df72125520f051-tgkkntbsmv.chromatic.com/?path=/docs/configure-your-project--docs

One is from `next`, the other one from my branch. A warning from React that the legacyRootAPI is in use doesn't appear in either of the deployed Storybooks. So our assumption, that this code was never executed or taken into account is actually true.